WebAssets/ts/tsconfig.json

```diff
@@ -3,7 +3,7 @@
         "."
     ],
     "compilerOptions": {
-        "target": "ES2018",
+        "target": "ESNext",

```

Comment:
see ES2019 earlier 😕

Reply:
This one is for publishing to npm (actually I do some minimal babel after that), which I expect people to transpile when they import. Other one (tests) must be able to run on node.